### PR TITLE
Stock image

### PR DESCRIPTION
**Pull request by colepalm**
Added a background image and made some edits to the main page accordingly.  Did some additional refactoring I had missed earlier on the login and change password pages.

### CODE REVIEW

**Comment by AaronHampshire** [comment]
Overall, I really like this. It certainly humanizes the site. A few thoughts on this. Feel free to disagree.

(1) I don't know if I love the picture. It's a bit too grainy. I think we should spend some money on a stock photo. We should probably be able to get one for under $10 and have dan grab the bill.
(2) When you keep resizing the window, I can get a guy with no dog, a dog with headless guy. It seems to shift around a bit. That might be changed with a different image, though.
(3) I like the white text on the header, and I think I would the text below white, too. 
(4) There's something off about the navbar black border. I'm not sure if it should be white, too? Or grey? Or not there at all? It might all change, though with a different picture. We may think black for everything.
(5) The text, "or create an account as a Realtor..." really gets lost depending on where it is on the picture (see number 2).

**Comment by AaronHampshire** [comment]
New image looks REALLY nice. I was assuming we would have people in it, but there's something really calming about it. And that's what we're trying to do with the app: bring some calm and structure to a chaotic, messy process.

**Comment by AaronHampshire** [comment]
The font color is really nice, too. I like the white below and grey above. Solid choices. 

I think we can make a leap here if we stretched the entire height. I think we can do that with:
`html, body { height: 100%; }` and adding the 100% height to your  bg-img ID. This will also allow us to pop content below it to introduce cool features. You can push back on that, but I'm a fan of the immersive landing page.

File: RealEstate/apps/core/templates/core/base.html [hunk 1]
```diff
@@ -21,67 +21,77 @@
...
-            <span class="sr-only">Toggle navigation</span>
-            <span class="icon-bar"></span>
-            <span class="icon-bar"></span>
-            <span class="icon-bar"></span>
-          </button>
-          <a class="navbar-brand" href="{% url 'home' %}">
-            <span class="glyphicon glyphicon-home" aria-hidden="true"></span> Real Estate App
-          </a>
-        </div>
+    {% if user.is_active %}
```
Comment (cdschmitz) [inline RealEstate/apps/core/templates/core/base.html]: Can you change the is_active checks to is_authenticated?  is_active is a check for whether or not the account is active, ie have they deactivated it or not.  is_authenticated checks if they are currently logged in.

**Comment by cdschmitz** [comment]
I like the image, good stuff.  Can you merge in the current dev branch so we can see what it looks now that our main landing page is also a signup page?  Theres also one other issue which is that the background image gets applied to all views where the user is not logged in, which includes homebuyer signup.  Maybe we want this behavior but presumably we just want the background image for the signup page.  So if necessary I will help reorganize the templates so the image will appear on the landing page only.

File: RealEstate/apps/core/templates/core/base.html (hunk 1 above)
Comment (colepalm) [inline RealEstate/apps/core/templates/core/base.html]: Definitely.

**Pull request merged**